open-apparel-registry/open-apparel-registry
# Update 'Claim a Facility'

## Overview

Adds some helper text to encourage contributors to send
additional facility documentation to use via email, to help speed
up the facility claim process.

Adds Green Button and Fairtrade USA to certification options and
HERhealth, HERfinance, and HERrespect to affiliation options in
Claim a Facility.

Connects #1029, #1315 

## Demo

<img width="797" alt="Screen Shot 2021-06-07 at 8 17 23 AM" src="https://user-images.githubusercontent.com/21046714/121190812-68b02f00-c839-11eb-8153-bbce9fd18972.png">

## Notes

Some related tasks requiring more complex migrations have been moved to #1384 

## Testing Instructions

* Run `./scripts/update` and `./scripts/server`
* Login as c2@example.com and select 'claim facility' for any facility
* On the fourth page of the claim form you should see the helper text
* Claim the facility
* Login as an admin and approve the claim
* Login as c2@example.com and navigate to the claim details
* Confirm that the additional fields appear in the dropdowns for certifications and affiliations

## Checklist
